python/tvm/autotvm/measure/measure_methods.py

```diff
@@ -150,33 +151,51 @@ def build(self, measure_inputs):
                             )
 
                         else:
-                            if "InstantiationError" in str(res.error):
-                                msg = str(res.error)
+                            tb, exception = res.error
```

Comment:
Better to have an assertion to make sure it is a tuple-2. Combining another comment I have, maybe it's easier to just let `res.error` be `tb + "\n" + str(ex)`.

Reply:
For `BuildResult` we need to maintain the backtrace separately so that we can check for it here. And I think we should make `MeasureResult` and `BuildResult` have the same structure for errors because it is less confusing. Now tuner.py concatenates the error with the traceback.